cmd/arcstat/arcstat

```diff
@@ -151,20 +155,37 @@ def usage():
 def kstat_update():
     global kstat
 
-    k = [line.strip() for line in open('/proc/spl/kstat/zfs/arcstats')]
+    if sys.platform.startswith('freebsd'):
+        k = sysctl.filter('kstat.zfs.misc.arcstats')
```

Comment:
Might be long enough to justify the following split:

```
def kstat_update():
  global kstat
  if sys.platform.startswith('freebsd'):
     kstat_update_freebsd(kstat)
  else:
     kstat_update_linux(kstat)
```

At least this helps readability.

Reply:
Thanks, I agree this should be refactored. It has been done and is queued for the next update to this PR.